source/runner/templates/_tests/__snapshots__/_bitbucketCloudTemplate.test.ts.snap

```diff
@@ -31,7 +31,7 @@ exports[`generating messages for BitBucket cloud Mixed icon messages match snaps
   
   
 
-  Generated by 🚫 [dangerJS](https://danger.systems/js) against e70f3d6468f61a4bef68c9e6eaba9166b096e23c
+  Generated by ὪB [dangerJS](https://danger.systems/js) against e70f3d6468f61a4bef68c9e6eaba9166b096e23c
```

Comment:
Are you sure this is going to be ok in both bitbucket cloud and bitbucket server, including the old bitbucket server versions that a lot of companies are still using?

Reply:
I do not have a bitbucket server env yet. But in current change. I replace the bitbucket cloud emoji with unicode emoji, which is supported by bitbucket server. I think it will be worked, may be not. I test the unicode in my bitbucket cloud account. The emoji can not be rendered. I will change this later.